src/main/java/com/google/api/generator/gapic/composer/common/AbstractServiceCallableFactoryClassComposer.java

```diff
@@ -191,7 +195,8 @@ protected MethodDefinition createGenericCallableMethod(
       String methodVariantName,
       List<Object> transportCallSettingsTemplateObjects,
       String callSettingsVariantName,
-      List<Object> callSettingsTemplateObjects) {
+      List<Object> callSettingsTemplateObjects,
+      List<AnnotationNode> annotations) {
```

Comment:
What about having a thin wrapper around this that doesn't take `annotations`? Then the invocations that don't need `annotations` won't need to pass in an empty list/

Reply:
I can do that if you prefer it that way.